src/osemgrep/language_server/Unit_LS.ml

```diff
@@ -222,29 +234,46 @@ let processed_run () =
   in
   pack_tests "Processed Run" tests
 
-let session_rules () =
+let ci_tests () =
   let with_ci_client =
     let make_fn (req : Cohttp.Request.t) body =
       ignore body;
+      let uri = req |> Cohttp.Request.uri |> Uri.path in
       Http_mock_client.check_method `GET req.meth;
+      let body_file =
+        match uri with
+        | "/api/agent/deployments/scans/config" ->
+            "./tests/ls/ci/rule_conf_resp.json"
```

Comment:
Again I think it would better to inline their content here, like let body reply weither with a File of Fpath.t or a Content of string

Reply:
Fair, will change